components/layout/webrender_helpers.rs

```diff
@@ -513,6 +508,8 @@ impl WebRenderDisplayItemConverter for DisplayItem {
                                     item.base.clip.to_clip_region(frame_builder),
                                     pipeline_id);
             }
+            DisplayItem::PushStackingContextClass(_) => {}
+            DisplayItem::PopStackingContextClass(_) => {}
```

Comment:
nit: Could use an or-pattern (`DisplayItem::PushStackingContextClass(_) | DisplayItem::PopStackingContextClass(_) => {}`) if you want, up to you